modules/appeals_api/app/workers/appeals_api/weekly_error_report.rb

```diff
@@ -11,11 +13,11 @@ class WeeklyErrorReport
     # Only retry for ~48 hours since the job is run weekly
     sidekiq_options retry: 16, unique_for: 48.hours
 
-    def perform(to: Time.zone.now, from: 1.week.ago.beginning_of_day)
```

Comment:
The `to` and `from` arguments weren't actually being used in the mailer, so there's no need to ask for them.